graphdatascience/session/aurads_sessions.py

```diff
@@ -54,9 +54,9 @@ def get_or_create(
         if existing_session:
             session_id = existing_session.id
             # 0MB is AuraAPI default value for memory if none can be retrieved
-            if existing_session.memory != "0MB" and existing_session.memory != memory.value:
+            if existing_session.memory != memory:
```

Comment:
Right thats the other reason why we dont parse.
Newly created instances will return `0MB` on the instance details endpoint. So this will now fail if you quickly call the function twice